pkgs/by-name/re/regolith/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "regolith";
+  version = "1.5.1";
+
+  src = fetchFromGitHub {
```

Comment:
Can we add an automatic package update script using `nix-update`? See [documentation](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#automatic-package-updates). Try to manually test it by first manually decreasing the version and running `nix-shell maintainers/scripts/update.nix --argstr package <package_name>`.

```nix
passthru.updateScript = nix-update-script { };
```